app/pathogen/sarscov2/layout.tsx

```diff
@@ -1,48 +1,40 @@
 import React from "react";
-import { SarsCov2Providers, Hydrate } from "@/contexts/sarscov2-context/sarscov2-context";
-import { dehydrate } from "@tanstack/query-core";
+import { Hydrate, dehydrate } from "@tanstack/react-query";
 import getQueryClient from "@/components/customs/getQueryClient";
+import { request } from 'graphql-request';
 import { notFound } from 'next/navigation'
+import { GenericPathogenPageLayout } from "../generic-pathogen-page-layout";
+import { SarsCov2Providers } from "@/contexts/pathogen-context/pathogen-contexts/sc2-context";
+import { sarsCov2Estimates } from "@/hooks/useNewSarsCov2Data";
+import { sarsCov2Filters } from "@/hooks/useSarsCov2Filters";
 
 export default async function ArboLayout({
   children,
 }: {
   children: React.ReactNode;
 }) {
   const queryClient = getQueryClient();
-  //TODO: this segment is technically repeated in useSarsCov2Data.tsx. It should be refactored to be DRY.
-  console.time("prefetchQuerySero");
+
   await queryClient.prefetchQuery({
-    queryKey: ["SarsCov2Records"],
-    queryFn: () =>
-    fetch(`${process.env.NEXT_PUBLIC_API_BASE_URL}/sarscov2/records`, { cache: 'no-store' }).then(
-        (response) => response.json(),
-      ),
+    queryKey: ["sarsCov2Estimates"],
+    queryFn: () => request(process.env.NEXT_PUBLIC_API_GRAPHQL_URL ?? '', sarsCov2Estimates)
   });

```

Comment:
Like I said, this breaks SC2Tracker ofc because it's not using the new data source. Just give me a bit here and we'll get it back as we rebuild, just be patient :)